src/datasets/other/serp_events/reference.md

```diff
@@ -430,73 +432,108 @@ Number of SERP impressions with ads loaded and an ad blocker in use:
-| `num_ads_notshowing`       | Inferred number of ads that are loaded but not visible, and not blocked by an ad blocker. For example, ads in the carousel that will be shown on clicking "Expand" button. Use this to count "ads that are available but not visible".                                              |
+The `v2` table has 1 row per SERP impression, each representing a single page load of a SERP.
+Most columns contain impression-level properties.
+There are also 3 array-valued columns listing impressions and engagements by component.
+
+| Column                                       | Description                                                                                                                                                                                                                                                  |
+| -------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `impression_id`                              | UUID identifying SERP page loads. Use `COUNT(DISTINCT impression_id)` to count unique SERP impressions when `CROSS JOIN UNNEST`ing the array-valued columns.                                                                                                 |
+| `ping_seq`                                   | `ping_info.seq` from the events ping. Use together with `event_timestamp` for event sequencing.                                                                                                                                                              |
+| `event_timestamp`                            | Glean event timestamp for the `serp.impression` event corresponding to the SERP page load.                                                                                                                                                                   |
+| `is_shopping_page`                           | Is the SERP a shopping page, resulting from clicking on the "Shopping" tab?                                                                                                                                                                                  |
+| `is_private`                                 | Was the SERP loaded while in Private Browsing Mode?                                                                                                                                                                                                          |
+| `is_signed_in`                               | Was the SERP loaded while signed into a search provider account?                                                                                                                                                                                             |
```

Comment:
I think it'd be clearer if these were statements rather than questions, IE for `is_shopping_page` something like "True when the SERP is a shopping page, resulting from clicking the "Shopping" tab".